app/src/pages/LandingPage/Section2/FeatureCard.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+import clsx from 'clsx'
+
+interface IFeatureCard {
+  title: string
+  backgroundColor: string
+  image: React.ReactNode
+  backgroundImage: string
+}
+
+interface IFeatureCardProps extends IFeatureCard {
+  index: number
+}
+
+function FeatureCard({ title, backgroundColor, image, index, backgroundImage }: IFeatureCardProps) {
+  const isEven = index % 2 === 0
+  return (
+    <div
+      className={clsx(
+        'w-full flex justify-around items-center h-[433px] border-black border-t-2 ',
+        isEven ? 'flex-row' : 'flex-row-reverse',
+      )}
+    >
+      <div
+        style={{ backgroundImage }}
```

Comment:
`style={{ backgroundImage: `url(${backgroundImagePath})` }}`. We could get the backgroundImagePath only because all of them use `url(...)`